src/users/users.service.ts

```diff
@@ -70,6 +71,20 @@ export class UsersService {
     return this.usersRepository.findOne({ email });
   }
 
+  async checkAdminPermissions(email: string): Promise<permissionDto> {
```

Comment:
Please rename to reflect its purpose

```suggestion
  async getPermissions(email: string): Promise<permissionDto> {
```